src/test/java/nl/topicus/mssql2monetdb/test/CopyTableTest.java

```diff
@@ -0,0 +1,48 @@
+package nl.topicus.mssql2monetdb.test;
+
+import static org.junit.Assert.*;
+import nl.topicus.mssql2monetdb.CopyTable;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CopyTableTest {
+
+	@Test
+	public void testGenerateCountQuery() {
+		CopyTable table = new CopyTable();
+		
+		// test standard count query based on table from name
+		table.setFromName("test");
+		Assert.assertEquals(table.generateCountQuery(), "SELECT COUNT(*) FROM [test]");
+		
+		table.setFromName(null);
+		
+		// test a custom count query
+		String customCountQuery = "SELECT COUNT(*) FROM myQuery WHERE bla = 'something'";
+		table.setFromCountQuery(customCountQuery);		
+		Assert.assertEquals(table.generateCountQuery(), customCountQuery);	
```

Comment:
Getters en setters hoef je idd niet te unit-testen. Deze test zit er een beetje tussenin omdat je hier (toevallig) altijd de inputwaarde weer als outputwaarde verwacht. Maar nog steeds geldt dat voor unittests in principe het verwachte resultaat nooit een relatie mag hebben met de input.